build.gradle.kts

```diff
@@ -11,4 +11,13 @@ allprojects {
 
     apply(plugin = "org.jetbrains.intellij")
     apply(plugin = "kotlin")
+
+    intellij {
+        type = "IC"
+        version = "2018.2"
```

Comment:
lets externalize this, in a separate PR. maybe an issue to track so we don't forget.

Reply:
#15